src/fileopener/plugin.ts

```diff
@@ -171,25 +171,21 @@ class FileOpener implements IFileOpener {
     for (let h of this._handlers) {
       if (h.fileExtensions.indexOf(ext) !== -1) handlers.push(h);
     }
+
     // If there was only one match, use it.
     if (handlers.length === 1) {
```

Comment:
Right.  I deleted that comment on the other PR because I realized that after I submitted it.